vue/components/ui/button-color-platforme/button-color-platforme.vue

```diff
@@ -40,48 +40,167 @@
     min-width: 160px;
 }
 
-.button-color.disabled {
-    cursor: default;
-    opacity: 0.4;
+.button-color.button-color-default.disabled {
+    background-color: #eceef1;
+    border: 2px solid #eceef1;

```

Comment:
If the border width (2px) and style (solid) is the same across all colors/states, then define it at the `button-color` level.

Then, you just have to change the `border-color` property on each state.